Refresh.Interfaces.APIv3/Endpoints/AuthenticationApiEndpoints.cs

```diff
@@ -107,6 +108,10 @@ public ApiResponse<IApiAuthenticationResponse> Authenticate(RequestContext conte
         
         context.Logger.LogInfo(BunkumCategory.Authentication, $"{user} successfully logged in through the API");
 
+        // Update pin progress for signing into the API
+        database.IncrementUserPinProgress((long)ManuallyAwardedPins.SignIntoWebsite, 1, user, false);
+        database.IncrementUserPinProgress((long)ManuallyAwardedPins.SignIntoWebsite, 1, user, true);
```

Comment:
Bleh, this sucks. Maybe that last param should be nullable, where null means update for both beta, not-beta?